src/lib/drivers/extension.js

```diff
@@ -41,7 +41,9 @@ class ExtensionDriver extends Driver {
       .then(tabId => {
         this._tabId = tabId;
         this.beginLogging();
-        return this.attachDebugger_(tabId);
+        return this.attachDebugger_(tabId)
+            .then(_ => this.enableRuntimeEvents())
+            .then(_ => tabId);
```

Comment:
is resolving on `tabId` necessary? It matches what was there before, but anything using `driver.connect()` and expected a response is going to be disappointed when running in the CLI